### DISCUSSION TITLE
hotkey

### DISCUSSION THREAD

**Discussion by teeseus** [Tampermonkey/tampermonkey#1332] [Ideas]
hotkey for quicker script creation would be great

**Comment by derjanb** [comment]
Simply go to `chrome://extensions/shortcuts` in Chrome or `about:addons` > `Manage extension shortcuts` in Firefox and assign a shortcut to `Open new script tab`.

![image](https://user-images.githubusercontent.com/767504/133831090-150ccf98-18c3-459d-9c1b-e74709e20d2b.png)

**Reply by teeseus** [reply]
ty :D

**Reply by teeseus** [reply]
how are these options implemented?
